components/pkg-export-kubernetes/defaults/KubernetesManifest.hbs

```diff
@@ -0,0 +1,36 @@
+---
+## Secret for initial configuration.
+#apiVersion: v1
+#kind: Secret
+#metadata:
+#  name: user-toml-secret
+#type: Opaque
+#data:
+#  # Each configuration item needs to be encoded in base64.
+#  # Plain text content of the secret: "port = 4444"
+#  user.toml: cG9ydCA9IDQ0NDQ=
+#---
```

Comment:
What's the intention of including this commented block?

Reply:
To make it easy to add an initial user.toml to the service for users not well versed in kubernetes. Maybe this could be later dropped in favor of some flag where you pass contents of the user.toml and it computes base64 value out of it and puts in the manifest.